docs/articles/documentation/guides/basic-guides/obtain-client-side-info.md

```diff
@@ -174,6 +174,15 @@ does not finish before the callback, suspend the test until the callback fires.
   >
   > The return value is the only way to obtain data from client functions.
 
+* You can not use iterable destructuring (e.g. [`spread`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), [`rest`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Rest_parameters) operators) on non-array objects (e.g. `NodeList`, `HTMLCollection`).
+
+* Some of the [newly added](https://tc39.es/ecma262/) JavaScript features do not work properly within client functions. These include, but are not limited to:  
```

Comment:
Maybe we can just write `These features include:`?

It literally means that there are other features. At the same time, we don't emphasize this fact.